src/modules/auth/index.js

```diff
@@ -170,6 +173,27 @@ export default class AuthModule extends Module {
 		});
 	}
 
+	logoutOnExpiredSession (that, al=false) {
+		const difference = moment(that.UserInfo.exp).valueOf() - moment.now();
+		let alert = al;
+
+		if (!that.UserInfo?.exp) {
+			setTimeout(that.logoutOnExpiredSession, 1000, that);
+		}
+
+		if (difference <= 0) {
+			that.logout();
+			return;
+		} else if (difference < 60000 && !alert) {
+			that.App.addAlert("warning", `Your session will expire in ${Math.floor(difference/1000)} seconds.`);
```

Comment:
This has to be localisable.

Reply:
We have an issue with localisation outside of functional react components right now so I suppose it should be done with localisation of Sidebar components

Reply:
And you cannot localize the message here?